postgres/datadog_checks/postgres/data/conf.yaml.example

```diff
@@ -1,74 +1,68 @@
+    ## @param port - integer - optional - default: 5432
+    ## The port to use when connecting to PostgreSQL.
     #
-    port: 5432
+    # port: 5432
 
-    ## @param user - string - required
-    ## Datadog Username created to connect to PostgreSQL.
+    ## @param username - string - required
+    ## The datadog Username created to connect to PostgreSQL.
     #
     username: datadog
 
-    ## @param pass - string - required
-    ## Password associated with the Datadog user.
+    ## @param password - string - optional
+    ## The password associated with the Datadog user.
     #
-    password: <UNIQUEPASSWORD>
+    # password: <PASSWORD>
 
     ## @param dbname - string - optional - default: postgres
-    ## Name of the PostgresSQL database to monitor.
+    ## The Name of the PostgresSQL database to monitor.
     ## Note: If omitted, the default system postgres database is queried.
     #
-    # dbname: <DB_NAME>
+    # dbname: <DBNAME>
 
-    ## @param ssl - string - optional - default: false
+    ## @param ssl - string - optional - default: False
     ## This option determines whether or not and with what priority a secure SSL TCP/IP connection
     ## is negotiated with the server. There are six modes:
-    ##
     ## - `disable`: Only tries a non-SSL connection.
     ## - `allow`: First tries a non-SSL connection; if if fails, tries an SSL connection.
     ## - `prefer`: First tries an SSL connection; if it fails, tries a non-SSL connection.
     ## - `require`: Only tries an SSL connection. If a root CA file is present, verifies the certificate in
-    ##            the same way as if verify-ca was specified.
+    ##              the same way as if verify-ca was specified.
     ## - `verify-ca`: Only tries an SSL connection, and verifies that the server certificate is issued by a
-    ##              trusted certificate authority (CA).
+    ##                trusted certificate authority (CA).
     ## - `verify-full`: Only tries an SSL connection and verifies that the server certificate is issued by a
-    ##                trusted CA and that the requested server host name matches the one in the certificate.
+    ##                  trusted CA and that the requested server host name matches the one in the certificate.
     ##
-    ## For a detailed description of how these options work, see https://www.postgresql.org/docs/current/libpq-ssl.html
+    ## For a detailed description of how these options work see https://www.postgresql.org/docs/current/libpq-ssl.html
     ##
     ## Note: `true` is an alias for `require`, and `false` is an alias for `disable`
     #
-    # ssl: false
-
-    ## @param query_timeout - integer - optional
-    ## Abort any statement that takes more than the specified number of milliseconds, starting from the time the command
-    ## arrives at the server from the client. A value of zero (the default) turns this off.
-    ## Cancelled queries won't log any metrics
-    #
-    # query_timeout: 1000
```

Comment:
Seems `query_timeout` is not present anymore.

Reply:
Oh, that's right cause it was added on a separate recent pr. I'll fix it